dev_tools/utils/nxdl_utils.py

```diff
@@ -260,59 +259,35 @@ def belongs_to_capital(params):
 
 def get_local_name_from_xml(element):
     """Helper function to extract the element tag without the namespace."""
-    return element.tag[element.tag.rindex("}") + 1 :]
+    return remove_namespace_from_tag(element.tag)
 
 
 def get_own_nxdl_child_reserved_elements(child, name, nxdl_elem):
```

Comment:
RHS and LHS function have the same name but return differently formatted pieces of information eventually even different pieces of information